testuseradd.py

```diff
@@ -0,0 +1,28 @@
+"""
+This test checks addition of user 
+"""
+import unittest
+from .testutils import system
+
+
+class UserAddtest(unittest.TestCase):
+    """
+    Useradd test
+    """
+
+    def test_useradd(self):
+        out, err, retcode = system("sudo useradd test_satyajit")
+        self.assertEqual(retcode, 0, out)
+        out, err, retcode = system("cat /etc/passwd")
```

Comment:
Instead of using `cat` use pythonic way :)
Just open the file and read and check whatever is required in pythonic way.